src/docs/docker-plugin.md

```diff
@@ -25,6 +26,8 @@ Create a config file `/etc/ndnfs/ndnfs.json` using this example:
 | nedgeport | Port of NexentaEdge REST API server| 8080 |  true |
 | username  | NexentaEdge REST API server user name| admin | true |
 | password  | NexentaEdge REST API server password | nexenta | true |
+| cluster   | NexentaEdge cluster name to reduce command length  |  | false |
+| forceBucketDeletion | Enforce bucket deletion during volume removement | false | false |
```

Comment:
On docker volume delete operation - the bucket will also be deleted

(is docker operation called volume delete or volume remove?)